### PR TITLE
{devel}[GCCcore/8.2.0] gdbgui v0.13.1.2 w/ Python 3.7.2

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)
depends on #8488

### CODE REVIEW

**Comment by Micket** [comment]
And In case someone requests asks; this package only supports python 3 (and I want to put GDB as a dep as well).

File: easybuild/easyconfigs/g/gdbgui/gdbgui-0.13.1.2-GCCcore-8.2.0-Python-3.7.2.eb [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+builddependencies = [('binutils', '2.31.1')]
+
+dependencies = [
+    ('Python', '3.7.2'),
+    ('GDB', '8.3', '-Python-3.7.2'),
+]
+
+exts_default_options = {
```
Comment (smoors) [inline easybuild/easyconfigs/g/gdbgui/gdbgui-0.13.1.2-GCCcore-8.2.0-Python-3.7.2.eb]: `exts_default_options` is not needed with `PythonBundle` easyblocks

File: easybuild/easyconfigs/g/gdbgui/gdbgui-0.13.1.2-GCCcore-8.2.0-Python-3.7.2.eb [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        'source_tmpl': '%(version)s.tar.gz',
+        'source_urls': ['https://github.com/cs01/gdbgui/archive/'],
+        'checksums': ['6d8eb296e664e4170ff063a3ad548bc7bd0badff04db59d825dcc5691e03730c'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/'],
+}
```
Comment (smoors) [inline easybuild/easyconfigs/g/gdbgui/gdbgui-0.13.1.2-GCCcore-8.2.0-Python-3.7.2.eb]: these are the default `sanity_check_paths` for `PythonBundle`, so not necessary to add.

File: easybuild/easyconfigs/g/gdbgui/gdbgui-0.13.1.2-GCCcore-8.2.0-Python-3.7.2.eb [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    }),
+    ('Flask-SocketIO', '2.9.6', {
+        'source_urls': ['https://pypi.python.org/packages/source/f/flask-socketio/'],
+        'checksums': ['f49edfd3a44458fbb9f7a04a57069ffc0c37f000495194f943a25d370436bb69'],
+    }),
+    ('Flask', '0.12.4', {
+        'source_urls': ['https://pypi.python.org/packages/source/f/flask/'],
+        'checksums': ['2ea22336f6d388b4b242bc3abf8a01244a8aa3e236e7407469ef78c16ba355dd'],
+    }),
+    ('Click', '7.0', {
```
Comment (smoors) [inline easybuild/easyconfigs/g/gdbgui/gdbgui-0.13.1.2-GCCcore-8.2.0-Python-3.7.2.eb]: `Click` is already included in Python-3.7.2

File: easybuild/easyconfigs/g/gdbgui/gdbgui-0.13.1.2-GCCcore-8.2.0-Python-3.7.2.eb [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+        'source_urls': ['https://pypi.python.org/packages/source/c/click/'],
+        'checksums': ['5b94b49521f6456670fdb30cd82a4eca9412788a93fa6dd6df72c94d5a8ff2d7'],
+    }),
+    ('Jinja2', '2.10.1', {
```
Comment (smoors) [inline easybuild/easyconfigs/g/gdbgui/gdbgui-0.13.1.2-GCCcore-8.2.0-Python-3.7.2.eb]: `Jinja2` is already included in Python-3.7.2

File: easybuild/easyconfigs/g/gdbgui/gdbgui-0.13.1.2-GCCcore-8.2.0-Python-3.7.2.eb [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    ('Werkzeug', '0.15.4', {
+        'source_urls': ['https://pypi.python.org/packages/source/w/werkzeug/'],
+        'checksums': ['a0b915f0815982fb2a09161cb8f31708052d0951c3ba433ccc5e1aa276507ca6'],
+    }),
+    ('socketio', '4.1.0', {
+        'source_tmpl': 'python-socketio-%(version)s.tar.gz',
+        'source_urls': ['https://pypi.python.org/packages/source/p/python-socketio/'],
+        'checksums': ['c7ffeac3d81f2d8d63b3ec7ed1e2d4478cc84aa1da666c1934c19432f4b8c0f8'],
+    }),
+    ('MarkupSafe', '1.1.1', {
```
Comment (smoors) [inline easybuild/easyconfigs/g/gdbgui/gdbgui-0.13.1.2-GCCcore-8.2.0-Python-3.7.2.eb]: is already included in Python-3.7.2

File: easybuild/easyconfigs/g/gdbgui/gdbgui-0.13.1.2-GCCcore-8.2.0-Python-3.7.2.eb [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+    }),
+    ('Flask-Compress', '1.4.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/f/flask-compress/'],
+        'checksums': ['468693f4ddd11ac6a41bca4eb5f94b071b763256d54136f77957cfee635badb3'],
+    }),
+    ('Flask-SocketIO', '2.9.6', {
+        'source_urls': ['https://pypi.python.org/packages/source/f/flask-socketio/'],
+        'checksums': ['f49edfd3a44458fbb9f7a04a57069ffc0c37f000495194f943a25d370436bb69'],
+    }),
+    ('Flask', '0.12.4', {
```
Comment (smoors) [inline easybuild/easyconfigs/g/gdbgui/gdbgui-0.13.1.2-GCCcore-8.2.0-Python-3.7.2.eb]: @Micket there is already an easyconfig for `Flask`. given that several other codes need it too, maybe its better to update it (and add Flask-Compress, Flask-SocketIO) and add it as dependency for gdbgui?

Comment (Micket) [reply easybuild/easyconfigs/g/gdbgui/gdbgui-0.13.1.2-GCCcore-8.2.0-Python-3.7.2.eb]: @smoors gdbgui unfortunately has a requirement for `Flask < 1.0` though
https://github.com/cs01/gdbgui/blob/master/setup.py#L12
as well as a requirement on an older Flask-SocketIO

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in this PR)
nic425 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/0789398e74ac4d3c6561aa05bd61e26a for a full test report.

**Comment by smoors** [comment]
Going in, thanks @Micket!

**Pull request merged**